pkg/cri/sbserver/container_create.go

```diff
@@ -697,13 +698,10 @@ func (c *criService) buildLinuxSpec(
 		customopts.WithOOMScoreAdj(config, c.config.RestrictOOMScoreAdj),
 		customopts.WithPodNamespaces(securityContext, sandboxPid, targetPid, uids, gids),
 		customopts.WithSupplementalGroups(supplementalGroups),
-		customopts.WithAnnotation(annotations.ContainerType, annotations.ContainerTypeContainer),
-		customopts.WithAnnotation(annotations.SandboxID, sandboxID),
-		customopts.WithAnnotation(annotations.SandboxNamespace, sandboxConfig.GetMetadata().GetNamespace()),
-		customopts.WithAnnotation(annotations.SandboxUID, sandboxConfig.GetMetadata().GetUid()),
-		customopts.WithAnnotation(annotations.SandboxName, sandboxConfig.GetMetadata().GetName()),
-		customopts.WithAnnotation(annotations.ContainerName, containerName),
-		customopts.WithAnnotation(annotations.ImageName, imageName),
+	)
+	specOpts = append(
+		specOpts,
+		podsandbox.DefaultCRIAnnotations(sandboxID, containerName, imageName, sandboxConfig, false)...,
```

Comment:
Since we use it for containers too, `podsandbox/` might not be the best place? Possibly `pkg/cri/annotations` (so it can be consumed from server, sbserver, and podsandbox)?

Reply:
I trialed another spot (that I'm blanking on) and hit a cyclic import but let me try here

Reply:
Okay this worked out fine, updated